pandas/core/generic.py

```diff
@@ -5661,22 +5661,59 @@ def clip_lower(self, threshold, axis=None, inplace=False):
         """
         Return copy of the input with values below given value(s) truncated.
 
+        It truncates values below a certain threshold. Threshold can be a single
+        value or an array, in the latter case it performs the truncation
+        element-wise.
+
         Parameters
         ----------
         threshold : float or array_like
+            Minimum value allowed. All values below threshold will be
+            set to this value.
         axis : int or string axis name, optional
             Align object with threshold along the given axis.
         inplace : boolean, default False
             Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+                .. versionadded:: 0.21.0.
 
         See Also
         --------
-        clip
+        clip : Return copy of input with values below and above thresholds truncated.
 
         Returns
         -------
         clipped : same type as input
+
+        Examples
+        --------
+        >>> s = pd.Series([5, 6, 7, 8, 9])
+        >>> s
+        0    5
+        1    6
+        2    7
+        3    8
+        4    9
+        dtype: int64
+
+        >>> s.clip_lower(4)
+        0    5
+        1    6
+        2    7
+        3    8
+        4    9
+        dtype: int64
+
+        >>> t = [4,8,7,2,5]
+        >>> t
+        [4, 8, 7, 2, 5]
+
+        >>> s.clip_lower(t)
+        0    5
+        1    8
```

Comment:
May be a sentence before this explaining what this example shows would be useful.